recipes-devtools/spirv-llvm-translator/spirv-llvm-translator/0001-cmake-allow-to-enable-disable-ccache.patch

```diff
@@ -0,0 +1,32 @@
+From 13b679eb76e6877b40fd41b91e71cb172bcf7b26 Mon Sep 17 00:00:00 2001
+From: Naveen Saini <naveen.kumar.saini@intel.com>
+Date: Thu, 4 Mar 2021 11:23:20 +0800
+Subject: [PATCH] cmake: allow to enable/disable ccache
+
+By default ccache is enabled.
+
+Upstream-Status: Submitted [https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/930]
+
+Signed-off-by: Naveen Saini <naveen.kumar.saini@intel.com>
+---
+ CMakeLists.txt | 3 ++-
+ 1 file changed, 2 insertions(+), 1 deletion(-)
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 0ddb55c..c7d0498 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -50,8 +50,9 @@ if(LLVM_SPIRV_BUILD_EXTERNAL)
+ 
+   message(STATUS "Found LLVM: ${LLVM_VERSION}")
+ 
++  option(CCACHE_ALLOWED "allow use of ccache" TRUE)
+   find_program(CCACHE_EXE_FOUND ccache)
+-  if(CCACHE_EXE_FOUND)
++  if(CCACHE_EXE_FOUND AND CCACHE_ALLOWED)
```

Comment:
lets rename this knob to `ENABLE_CCACHE`

Reply:
PR already approved ! 
https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/930

Reply:
nice